cmd/gaia/init/genesis_accts.go

```diff
@@ -16,11 +16,11 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth"
 )
 
-// AddGenesisAccountCmd returns add-genesis-account cobra Command
+// AddGenesisAccountCmd returns add-genesis-account cobra Command.
 func AddGenesisAccountCmd(ctx *server.Context, cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "add-genesis-account [address_or_key_name] [coin][,[coin]]",
-		Short: "Add genesis account to genesis.json",
+		Short: "Add genesis account to a genesis file",
```

Comment:
I'd second this new wording if were to support an extra optional flag/argument to provide the genesis.json; needless to say that it should default to `gaiad` default genesis file.

Reply:
I think what @alessio is saying is that there is no way for this command to specify what the particular file path to genesis is, so `to a genesis file` is maybe overselling.